blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/Containers.java

```diff
@@ -15,7 +15,133 @@
 
     public abstract static class AbstractContainer<T extends Resource> extends Container<T> {
     }
+    
+    /**
+     * Combines multiple {@link Iterable} to treat them as a single one
+     *
+     * @param <T> iterable type
+     */
+    public static class CombinedIterable<T> implements Iterable<T> {
```

Comment:
@vivek yes, I recall and we discussed this, just pointing out that there is already code to handle this case.